library/Fission/Web/App/Update.hs

```diff
@@ -23,16 +27,18 @@ type API
   :> PatchAccepted '[JSON] NoContent
 
 update ::
-  ( MonadLogger  m
-  , MonadThrow   m
-  , MonadTime    m
-  , App.Modifier m
+  ( MonadLogger     m
+  , MonadThrow      m
+  , MonadTime       m
+  , MonadRemoteIPFS m
+  , App.Modifier    m
   )
   => Authorization
   -> ServerT API m
 update Authorization {about = Entity userId _} url newCID copyDataFlag = do
   now <- currentTime
-  Web.Error.ensureM $ App.setCID userId url newCID copyFiles now
+  size <- Web.Error.ensureM $ IPFS.Stat.getSizeRemote newCID
```

Comment:
Yeah, this one (comment above). Why not make `getSizeRemote` `User.getSize` and `App.getSize`, which reference these?

Reply:
I like the use of `ensureM`, though 👍